### ISSUE TITLE
[Design] Investigate non-text color contrast

### ISSUE THREAD

**Issue by aagonzales** [carbon-design-system/carbon#5154]
In WCAG 2.1 new non-text color contrast rules were made and we need to account for them.
https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html

- [ ] Investigate and document new non-text color contrast rules as applied to out system
- [ ] Audit component for area's where we fail new rules.
- [ ] Create solutions for non-text contrast problems and document solutions
